react/store/createNewRMA/components/RenderConditionDropdown.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { useIntl, defineMessages } from 'react-intl'
+import { Dropdown } from 'vtex.styleguide'
+
+const messages = defineMessages({
+  conditionNewWithBox: {
+    id: 'store/return-app.return-order-details.dropdown-conditions.new-with-box',
+  },
+  conditionNewWithoutBox: {
+    id: 'store/return-app.return-order-details.dropdown-conditions.new-without-box',
+  },
+  conditionUsedWithBox: {
+    id: 'store/return-app.return-order-details.dropdown-conditions.used-with-box',
+  },
+  conditionUsedWithoutBox: {
+    id: 'store/return-app.return-order-details.dropdown-conditions.used-without-box',
+  },
+  conditionSelectCondition: {
+    id: 'store/return-app.return-order-details.dropdown-conditions.placeholder.select-condition',
+  },
+})
+
+export const RenderConditionDropdown = ({ condition, handleCondition, id }) => {
+  const { formatMessage } = useIntl()
+  const conditionOptions = [
+    {
+      value: 'New With Box',
```

Comment:
Just to keep in mind, the value here will need to match the `enum` in the schema. Once we merge it all, we can handle it.